core/tests/protractor_utils/editor.js

```diff
@@ -430,25 +497,27 @@ var _selectRule = function(ruleElement, interactionId, ruleName) {
       optionElements[0].click();
     });
   });
+};
+
+// Before using this function ensure that expectValueToBe
+// (as declared in TODO below) is implemented in your parameter type editor.
+var expectRuleParametersToBe = function(interactionId, ruleName) {
```

Comment:
Checks that the current rule parameter values match the given ones. Note: the expected rule parameter values should be specified as additional arguments after the ruleName.

Before using this function, ensure that expectValueToBe (see the TODO below) is implemented in the corresponding parameter type editor.